### ISSUE TITLE
Smart Output consuming its inputs but not producing output.

### ISSUE THREAD

**Issue by Zarepheth** [gkbm2013/tinker-IO#32]
I have a Tinker's Construct smeltery. The smeltery works fine and previously the Smart Output also worked fine. But over the past week, I've had numerous cases where the Smart Output takes in the melted metals but fails to produce any output.

I've melted Ardite and Iron that I know did went into the Smart Outputs, but never came out.

What I'm currently seeing is the stream of molten materials going into the Smart Output. The Smart Output claims to be holding 6mb (or sometimes 12 mb) of molten material. Looking at the Smart Output, I never see the internal tank/buffer fill with anything. I never see the flash as ingots are being made. The hopper under the Smart Output and the subsequent hopper ducts and Storage Drawers storage network never see the ingots that should have come from the Smart Output. However, the Smeltery's liquid level decreases as the Smart Output were actually producing something. So I'm losing what I put into the Smeltery and not getting anything out.

On a side note: I can't seem to get the Tinkers Construct molten glass to form Glass Panes in the Smart Output. I either have to toss the molten glass in the Smart Output or install the Casting Basin upgrade. (A rather expensive upgrade, too; though it's a nice feature with the new version.)

Any suggestions for how to get the Smart Output to produce the ingots it ought to produce? Anything I should do to get you some useful logs? (The game hasn't crashed because of this, so there's no crash log.)

**Comment by Zarepheth** [comment]
Forgot to mention:
Minecraft 1.10.2
Forge 2185
TConstruct 1.10.2 - 2.6.1
tinker_io 1.10.2 release 2.4.1
Hopper Ducts 1.9.4-1.4.8
StorageDrawers 1.10.2-3.5.12 (note: the problem has existed even with versions of SD back to 3.5.8)

**Comment by Zarepheth** [comment]
An update: after saving my world and closing Minecraft, then restarting Minecraft and reloading my world, the partially filled Smart Output indicated 2 blocks and 5 ingots of molten iron (rather than the 6 mb prior to shutting down). And putting an ingot cast into the Smart Output caused it to start making ingots.

So, one work-around for the problem is to save the world and quit Minecraft, then restart Minecraft and reload the world. I was within a few blocks of the Smeltery and Smart Output at the time I saved and shutdown - if that makes any difference. I had tried teleporting a few thousand blocks away and teleporting back to see if Chunk loading/unloading might fix the problem. It once seemed to fix the issue 2-3 days ago.

The Smart Output, with no cast, still fails to make metal plates or glass panes.
